ADMINS.md

```diff
@@ -0,0 +1,7 @@
+## Admins
+
+| Admin           | GitHub ID                               | Affiliation |
+| --------------- | --------------------------------------- | ----------- |
+| Henri Yandell   | [hyandell](https://github.com/hyandell) | Amazon      |
+
+[This document](https://github.com/opensearch-project/.github/blob/main/ADMINS.md) explains what admins do in this repo. and how they should be doing it. If you're interested in becoming a maintainer, see [MAINTAINERS](MAINTAINERS.md). If you're interested in contributing, see [CONTRIBUTING](CONTRIBUTING.md).
```

Comment:
Does it explain what they do? 🤔

Reply:
Yup: `As an admin you own stewartship of the repository and its settings. Admins have admin-level permissions on a repository. Use those privileges to serve the community and protect the repository as follows.`